### ISSUE TITLE
Table min-width causes clipped text when generating PDFs

### ISSUE THREAD

**Issue by alex-ketch** [stencila/thema#9]
The `min-width` size seems to be larger than the PDF [page size we have defined in Encoda](https://github.com/stencila/encoda/blob/188e332880314cc64fae0b88366d71ac673d0f9e/src/codecs/pdf/index.ts#L46-L53).

https://github.com/stencila/thema/blob/e7773de6315bd049a002325efc57f69856e53835/src/themes/stencila/styles.css#L314

This causes the generated PDF to have clipped text as seen in screenshot below:

![Screen Shot 2019-08-01 at 17 01 10](https://user-images.githubusercontent.com/1646307/62328737-e0fd7a80-b481-11e9-9386-6edc8ee5fee6.png)

The `min-width` should be changed to `100%`, and the current value moved down to be inside the `@media-query` section near the end of the file.

**Comment by jlbrewe** [comment]
Hi! I have a project for a software methodology class to contribute to an open source repo and I am very interested in this project. Could I be assigned to this issue?

**Comment by alex-ketch** [comment]
Hi and thanks for your interest in contributing to this project @jlbrewe, we really appreciate it!
I was trying to reproduce this issue to provide you with some test files, but it seems like it got fixed somewhere along the way since 2019.

If you are still interested though, I've outlined [another issue here](https://github.com/stencila/thema/issues/310) that we’d love to get your help with. It does depend on a fix in another one of our projects though, which I can either provide you with guidance on where the fix would need to happen, or let you know once it's been fixed—which shouldn't take more than a day or two for me to get around to.
